docs/source/preview_pandas_datasets.md

```diff
@@ -0,0 +1,74 @@
+# Preview Tabular Data in Kedro-Viz
+
+We use the {doc}`spaceflights tutorial<kedro:tutorial/spaceflights_tutorial>` to demonstrate how to add data preview for the `customer`, `shuttle` and `reviews` datasets. Even if you have not yet worked through the tutorial, you can still follow this example; you'll need to use the Kedro starter for the spaceflights tutorial to generate a copy of the project with working code in place.
+
+If you haven't installed Kedro {doc}`follow the documentation to get set up<kedro:get_started/install>`. 
+
+```{important}
+We recommend that you use the same version of Kedro that was most recently used to test this tutorial (0.19.0). To check the version installed, type `kedro -V` in your terminal window.
+```
+
+In your terminal window, navigate to the folder you want to store the project. Generate the spaceflights tutorial project with all the code in place by using the [Kedro starter for the spaceflights tutorial](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pandas):
+
+
+```bash
+kedro new --starter=spaceflights-pandas
+```
+
+When prompted for a project name, you can enter anything, but we will assume `Spaceflights` throughout.
```

Comment:
```suggestion
When prompted for a project name, you can enter anything, but we will assume Spaceflights throughout.
```

Reply:
not sure about this one, because we are mentioning as project_name.